bloom_nofos/nofos/api/schemas.py

```diff
@@ -0,0 +1,74 @@
+from ninja import ModelSchema, Schema
+from typing import List, Optional
+from nofos.models import Nofo, Section, Subsection
+
+
+class SubsectionSchema(ModelSchema):
+    class Config:
+        model = Subsection
+        model_fields = [
+            "name",
+            "html_id",
+            "order",
+            "tag",
+            "body",
+            "callout_box",
+            "html_class",
+        ]
+        model_fields_optional = ["html_class"]  # Fields that should have defaults
+
+
+class SectionSchema(ModelSchema):
+    subsections: List[SubsectionSchema]
+
+    class Config:
+        model = Section
+        model_fields = ["name", "html_id", "order", "has_section_page"]
+
+
+class NofoSchema(ModelSchema):
+    sections: List[SectionSchema]
+
+    class Config:
+        model = Nofo
+        model_fields = [
+            "title",
+            "filename",
```

Comment:
Let's also return "id".

The section and subsection ids don't matter (we would never return a subsection on its own) but I feel like NOFO ID is a useful handle so that you know how to find it again.